vips/vip-171/vip-171-testnet.ts

```diff
@@ -0,0 +1,297 @@
+import { ethers } from "hardhat";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const NEW_VBEP20_DELEGATE_IMPL = "0xAC5CFaC96871f35f7ce4eD2b46484Db34B548b40";
+const ACCESS_CONTROL_MANAGER = "0x45f8a08F534f34A97187626E05d4b6648Eeaa9AA";
+const NORMAL_TIMELOCK = "0xce10739590001705F7FF231611ba4A48B2820327";
+const FAST_TRACK_TIMELOCK = "0x3CFf21b7AF8390fE68799D58727d3b4C25a83cb6";
+const CRITICAL_TIMELOCK = "0x23B893a7C45a5Eb8c8C062b9F32d0D2e43eD286D";
+const PROTOCOL_SHARE_RESERVE = "0x25c7c7D6Bf710949fD7f03364E9BA19a1b3c10E3";
+const COMPTROLLER = "0x94d1820b2D1c7c7452A163983Dc888CEC546b77D";
+
+interface AssetConfig {
+  name: string;
+  address: string;
+  reduceReservesBlockDelta: number;
+  acceptAdmin: boolean;
+  isMock: boolean; // Defines underlying is MockToken
+}
+
+export const CORE_MARKETS: AssetConfig[] = [
+  {
```

Comment:
I would include the missing testnet markets:

* vBUSD: 0x08e0A5575De71037aE36AbfAfb516595fE68e5e4
* vTRXOLD: 0x369Fea97f6fB7510755DCA389088d9E2e2819278
* vTUSDOLD: 0x3A00d9B02781f47d033BAd62edc55fBF8D083Fb0
* vDOGE: 0xF912d3001CAf6DC4ADD366A62Cc9115B4303c9A9
* vMATIC: 0x3619bdDc61189F33365CC572DF3a68FB3b316516
* vCAKE: 0xeDaC03D29ff74b5fDc0CC936F6288312e1459BC6
* vADA: 0x37C28DE42bA3d22217995D146FC684B2326Ede64
* vAAVE: 0x714db6c38A17883964B68a07d56cE331501d9eb6

11 (already included) + 8 (new ones) + UST,LUNA,BNB = 22

Reply:
Added [af09fb0](https://github.com/VenusProtocol/vips/pull/67/commits/af09fb00273c1ff846a5c74d45c6c17b0d6216b9)